features/support/xcode.rb

```diff
@@ -2,12 +2,21 @@
 
 module Xcode
   include MiniTest::Assertions
-  
+
   def self.product_with_name(name, destination:, derived_data_path:)
     glob = File.join(derived_data_path, "**/Build/**/Products/#{destination}/#{name}/")
     Dir.glob(glob).max_by { |f| File.mtime(f) }
   end
 
+  def self.info_plist_for_product_with_name(product:, destination:, derived_data_path:)
+    product_path = product_with_name(
+      product,
+      destination: destination,
+      derived_data_path: derived_data_path
+    )
+    File.join(product_path, "Info.plist")
```

Comment:
This is not true for macOS apps 😬. If I'm not wrong, I think it's under `Contents/Info.plist`